pnpm-workspace.yaml

```diff
@@ -7,35 +7,35 @@ packages:
   - test
 
 catalog:
-  '@platforma-sdk/block-tools': ^2.5.1
-  '@platforma-sdk/model': ^1.20.27
-  '@platforma-sdk/ui-vue': ^1.20.30
+  '@platforma-sdk/block-tools': ^2.5.3
+  '@platforma-sdk/model': ^1.21.20
+  '@platforma-sdk/ui-vue': ^1.21.21
   '@platforma-sdk/tengo-builder': ^1.17.5
-  '@platforma-sdk/workflow-tengo': ^2.9.5
+  '@platforma-sdk/workflow-tengo': ^2.9.6
 
-  '@platforma-sdk/test': ^1.20.27
-  '@milaboratories/graph-maker': ^1.1.18
-  '@milaboratories/software-pframes-conv': ^2.1.2
+  '@platforma-sdk/test': ^1.21.20
+  '@milaboratories/graph-maker': ^1.1.21
+  '@milaboratories/software-pframes-conv': ^2.1.5
   '@platforma-open/milaboratories.software-small-binaries': ^1.15.6
   '@platforma-open/milaboratories.software-subread': ^1.0.0
   '@platforma-open/milaboratories.software-star': ^1.0.1
   '@platforma-open/milaboratories.star-read-mapping.genome-assets': ^1.1.6
-  '@platforma-open/milaboratories.runenv-r': ^1.2.10
-  '@platforma-sdk/package-builder': ^2.14.2
+  '@platforma-open/milaboratories.runenv-r': ^1.2.11
+  '@platforma-sdk/package-builder': ^2.15.0
 
   '@milaboratories/helpers': ^1.6.11
 
   'ag-grid-enterprise': &ag-grid ^33.0.3
   'ag-grid-vue3': *ag-grid
 
   'vue': ^3.5.13
-  'vue-tsc': ^2.1.10
+  'vue-tsc': ^2.2.0
 
-  'typescript': ~5.6.3
+  'typescript': ^5.7.2
```

Comment:
vue-tsc update was released, and we finally can upgrade to the latest typescript